src/app.js

```diff
@@ -87,17 +87,18 @@ app.use(
 )
 
 const imgSrc = [
-  "'self'"
+  "'self'",
+  'https://www.googletagmanager.com', // Support GA4 per https://developers.google.com/tag-platform/tag-manager/web/csp
+  'https://firefoxusercontent.com',
+  'https://mozillausercontent.com/',
+  'https://monitor.cdn.mozilla.net/'
```

Comment:
Y'all still using the CDN?
I thought all the logos were local now, like: `https://stage.<sshhhh>/images/logo_cache/kickstarter.com.ico`

These were the only CDN results I found for a quick grep:
```sh
git grep -n "https://monitor.cdn.mozilla.net/" | cat

.env-dist:98:EDUCATION_VIDEO_URL_RELAY=https://monitor.cdn.mozilla.net/videos/FF_Relay_version_02.mp4
.env-dist:99:EDUCATION_VIDEO_URL_VPN=https://monitor.cdn.mozilla.net/videos/Mozilla_VPN.mp4
routes/email-l10n.js:15:    imgSrc: ["'self'", 'https://monitor.cdn.mozilla.net/'],
server.js:135:  'https://monitor.cdn.mozilla.net/'
server.js:168:        'https://monitor.cdn.mozilla.net/'
src/app.js:94:  'https://monitor.cdn.mozilla.net/'
```

&hellip; but wasn't sure if you're still using those education MP4s and the only mention in src/ seems to be this file. No big deal, but fewer exceptions the better, I s'pose.

Reply:
But if it fixes this, I'm a happy lad:

<img width="661" alt="Screenshot 2023-03-23 at 6 37 12 PM" src="https://user-images.githubusercontent.com/557895/227402358-2750d869-2d87-44ac-a0d3-4f549b259257.png">

Reply:
Ah -- actually that's a different issue.  FxA expires after 24hrs (don't quote me on the interval) and we don't handle it properly.  We have multiple tickets filed against this 😅

Reply:
signing out and signing back in should result in a "defined" session?